index.html

```diff
@@ -552,7 +552,42 @@ <h2>Creators</h2>
         <div class="container">
           <hr size=3px width=100% color="#3D4250">
           <h2>Join us</h2>
-          <p>If you have any questions or would like to become a part of our community, please <a href="https://github.com/move-fast-and-break-things/movefastandbreakthings.club/issues" target="_blank">follow the link</a> to open a new issue. Kindly describe why you are interested in joining our community.</p>
+          <div class="Join_Us_text"><p class="Join_Us_lines">Hi!</p><br>
+
+            Welcome to Move Fast and Break Things: a community where passionate engineers collaborate, experiment, and grow together – fast. <br>
+            
+            <p class="Join_Us_lines"><b>Who are we?</b> We are a group of motivated, self-organized, hardworking, and kind people of very different levels of experience who are passionate about software engineering and product building. Among our members are experienced engineers and leaders such as Ivan Perkhun (<a href="https://t.me/perhun">Lead DS at BUDU</a>), Irina Kogai (<a href="https://www.linkedin.com/in/ikogai/">Engineering Lead at YouTalk</a>), and Yurij Mikhalevich (<a href="https://www.linkedin.com/in/yurijmikhalevich/">Staff AI Engineering Lead at QA Wolf</a>).<br></p>
```

Comment:
Давай тут и далее всем внешним ссылкам (кроме email-ссылки) добавим ` target="_blank"`, чтобы они открывались в новой вкладке?

```suggestion
            <p class="Join_Us_lines"><b>Who are we?</b> We are a group of motivated, self-organized, hardworking, and kind people of very different levels of experience who are passionate about software engineering and product building. Among our members are experienced engineers and leaders such as Ivan Perkhun (<a href="https://t.me/perhun" target="_blank">Lead DS at BUDU</a>), Irina Kogai (<a href="https://www.linkedin.com/in/ikogai/" target="_blank">Engineering Lead at YouTalk</a>), and Yurij Mikhalevich (<a href="https://www.linkedin.com/in/yurijmikhalevich/" target="_blank">Staff AI Engineering Lead at QA Wolf</a>).<br></p>
```